documentation/Database-Schema-Application.md

```diff
@@ -0,0 +1,84 @@
+# Database Schema Application
+
+![Database Schema](images/database-schema-application.png)
+
+# Database Area description: Application
+
+## Application instance
+
+The application has all the visualisation and logic defined in the **application template** [(described in Database Schema Template)](Database-Schema-Template.md). When the Applicant clicks to apply for an application type (aka template) a new instance of one application is created for them. The Applicant user can make changes until submitting the application to be reviewed.
+
+### application
+
+Representation of the application instance. All nested elements are accessible via joined tables and can be created or queried in the same call using the GraphQL engine.
+
+The `template_id` links to the template used to display all sections and elements (questions) to the user.
+
+The `user_id` links to the Applicant user that created this application.
+
+The `serial` is a number used to display the application number to users.
+
+The `name` will be deduced by the template name + user + company names.
+
+The `is_active` is set to `'true'` for all applications that are still in the review process or are in Draft status.
+
+The `is_active` is set to `'false'` for applications that have finished the review process, have expired or were withdrawn by the Applicant.
+
+The `trigger` is updated everytime the application has changes done by users or a scheduler. See more about [triggers](Triggers-and-Actions.md)
+
+The `outcome` is either `'Pending'`, `'Approved'`, `'Rejected'`. The application is 'Pending' during the review process, each stage will store also the outcome, so it will only be updated to here once the review process is finished resulting in either 'Approved' or 'Rejected'.
+
+**To be considered:**
+
+- The `unique_identifier` is a new field. What is it used for? Should is replace the `serial`?
+
+### application_section
+
+Sections of the application. It will point to the `template_section` that stores all the elements (questions) to be displayed. We required to have this records to have those associated to reviews, which are done per section of each application.
```

Comment:
> We required to have this records to have those associated to reviews

Can't quite make sense of this. Looks like you might have left too many words in. ;)

Reply:
Hahaha The classic too many words for a sentence. You know that Tolstoi really cared about that. All his sentences are well thought to have the minimum number of words required. : )